code/game/objects/items/weapons/implants/implant.dm

```diff
@@ -569,5 +577,34 @@ the implant may become unstable and either pre-maturely inject the subject or si
 	else
 		return 0
 
-/obj/item/weapon/implant/peace/get_data()
-	return imp_data
+
+/obj/item/weapon/implant/holy
+	name = "holy implant"
+	desc = "Submits its user to the chants of a thousand chaplains."
+
+/obj/item/weapon/implant/holy/get_data()
+	var/dat = {"
+<b>Implant Specifications:</b><BR>
+<b>Name:</b> Holy Dogmatic Interference Implant<BR>
+<b>Life:</b> Anywhere from ten days to ten years depending on the strain placed upon the implant by the subject.<BR>
+<b>Important Notes:</b> This device was commissioned by Nanotrasen after it proved able to distract occult practitioners, making the unable to practice their dark arts.<BR>
+<HR>
+<b>Implant Details:</b><BR>
+<b>Function:</b> Submits its subject to the chants of a thousand chaplains.<BR>
+<b>Special Features:</b> Prevents cultists from using their runes and talismans, or from being the target of some of their peers' rituals.<BR>
+<b>Integrity:</b> Implant anchors itself inside the subject's bones to prevent blood pressure induced ejections."}
+	return dat
+
+/obj/item/weapon/implant/holy/implanted(mob/M)
+	if(!iscarbon(M))
+		return 0
+	var/mob/living/carbon/H = M
+	H << sound('sound/ambience/ambicha1.ogg')
+	H << sound('sound/ambience/ambicha2.ogg')
+	H << sound('sound/ambience/ambicha3.ogg')
+	H << sound('sound/ambience/ambicha4.ogg')
+	if(iscultist(H))
+		to_chat(H, "<span class='danger'>You feel uneasy as you suddenly start hearing a cacopony religious chants. You find yourself unable to perform any ritual.</span>")
+	else
+		to_chat(H, "<span class = 'notice'>You hear the millenia-old soothing gregorian chants of the clergy.</span>")
```

Comment:
soothing millennia-old Gregorian chants (capitalize, change order of adjectives)